core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -180,13 +200,21 @@ func (c *coordinator) ReportBlocks(
 	maxBlocks, // TODO: unused for now
 	maxCallbacks int, // TODO: unused for now
 ) (blocks []ocr2vrftypes.Block, callbacks []ocr2vrftypes.AbstractCostedCallbackRequest, err error) {
+	// Instantiate the gas used by this batch.
+	currentBatchGasLimit := coordinatorOverhead
+
 	// TODO: use head broadcaster instead?
 	currentHeight, err := c.lp.LatestBlock(pg.WithParentCtx(ctx))
 	if err != nil {
 		err = errors.Wrap(err, "header by number")
 		return
 	}
 
+	// Evict expired items from the cache.
+	now := time.Now().Unix()
```

Comment:
Similar to my previous comment, prefer `time.Now().UTC()` to stay timezone neutral.

Reply:
Post-convo I remembered that Unix time is timezone-neutral. I can make this 
`time.Now().UTC().Unix()` but it's not necessary.